### PR TITLE
Update Conclusion

### PR DESCRIPTION
**Pull request by jkrvivian**

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -554,7 +554,12 @@ \subsubsection{Messages Publishment}
...
+
+In the subscription-based data trading, there are some issues that worth to discuss and improve. Firstly, reduce the Gas usage of trading models, as we mentioned in Section~\ref{smart_contract_evaluation}, the key exchange process consumes a lot Gas due to the frequent store operation. Therefore, it is worth studying how zero-knowledge protocol could used to exchange key with lower cost, and even validate the key within transfers. Second, brokers are irreplaceable either from the view of architecture design or trading procedures, they are the key to the stable system. However, the issues such as the minimum number of brokers needed to operation the system, and the evaluation of adopting gossip protocol among brokers to gossipped trading status, which enables other brokers to handle trading process when on crashed.
```
Comment (jserv) [inline paper.tex]: 改寫為 It is worth discussing that ...

File: paper.tex [hunk 2]
```diff
@@ -556,7 +556,7 @@ \subsubsection{Messages Publishment}
...
+In the subscription-based data trading, there are some issues that worth to discuss and improve. Firstly, reduce the Gas usage of trading models, as we mentioned in Section~\ref{smart_contract_evaluation}, the key exchange process consumes a lot Gas due to the frequent store operation. Therefore, it is worth discussing that how zero-knowledge protocol could used to exchange key with lower cost, and even validate the key within transfers. Second, brokers are irreplaceable either from the view of architecture design or trading procedures, they are the key to the stable system. However, the issues such as the minimum number of brokers needed to operation the system, and the evaluation of adopting gossip protocol among brokers to gossipped trading status, which enables other brokers to handle trading process when on crashed.
```
Comment (howjmay) [inline paper.tex]: that worth discussing and improving

https://www.facebook.com/605863386172088/photos/a.629518213806605/750541168370975/?type=1&theater

Comment (howjmay) [inline paper.tex]: the key exchange process consumes a lot of Gas due to

Comment (howjmay) [inline paper.tex]: could be used to exchange key with lower cost

Comment (howjmay) [inline paper.tex]: Second, brokers are irreplaceable either from the view of architecture design or trading procedures, so they are the key to the stable system.

**Review by howjmay — CHANGES_REQUESTED** [review]

**Pull request merged**